crypto/err/err.c

```diff
@@ -408,25 +408,6 @@ void ERR_put_error(int lib, int func, int reason, const char *file, int line)
 {
     ERR_STATE *es;
 
-#ifdef _OSD_POSIX
-    /*
-     * In the BS2000-OSD POSIX subsystem, the compiler generates path names
-     * in the form "*POSIX(/etc/passwd)". This dirty hack strips them to
-     * something sensible. @@@ We shouldn't modify a const string, though.
-     */
-    if (strncmp(file, "*POSIX(", sizeof("*POSIX(") - 1) == 0) {
-        char *end;
-
-        /* Skip the "*POSIX(" prefix */
-        file += sizeof("*POSIX(") - 1;
-        end = &file[strlen(file) - 1];
-        if (*end == ')')
-            *end = '\0';
-        /* Optional: use the basename of the path only. */
-        if ((end = strrchr(file, '/')) != NULL)
-            file = &end[1];
-    }
-#endif
```

Comment:
The reason you're removing this?  Please do note that we still have a config target for that system:

https://github.com/openssl/openssl/blob/7f02a0932c8c182bc2f6320748d81e86d650c1a6/Configurations/10-main.conf#L1240-L1251

So if you're suggesting that system should be cleaned away, I think that should be done in a separate PR.

Reply:
Okay separate PR in #9185